melanie-lab/src/component/landing/_landing.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../style/main.scss';
```

Comment:
I don't think you need to import main in here, it should trickle down. Give it a try?

Reply:
gotcha. Wasn't thinking about it from the vars standpoint. I was wondering the same the other day. Well, tomorrow is SASS day, right?